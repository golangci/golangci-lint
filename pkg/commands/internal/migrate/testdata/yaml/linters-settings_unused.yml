issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  unused:
    field-writes-are-uses: false
    post-statements-are-reads: true
    exported-fields-are-used: false
    parameters-are-used: false
    local-variables-are-used: false
    generated-is-used: false
