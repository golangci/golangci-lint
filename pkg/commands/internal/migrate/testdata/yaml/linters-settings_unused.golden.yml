version: "2"
linters:
  settings:
    unused:
      field-writes-are-uses: false
      post-statements-are-reads: true
      exported-fields-are-used: false
      parameters-are-used: false
      local-variables-are-used: false
      generated-is-used: false
